pytrip/lib/core.c

```diff
@@ -822,6 +877,129 @@ static PyObject* function_ranges(PyObject *self, PyObject *args){
 
 }
 
+static PyObject* binary_search_intersection(PyObject *self, PyObject *args){
```

Comment:
add comment what this function does

Reply:
and also comment function `function_ranges`

Reply:
no.